diff --git a/hal/hal_intf.c b/hal/hal_intf.c
index 38e306b..a5cd2d4 100644
--- a/hal/hal_intf.c
+++ b/hal/hal_intf.c
@@ -19,11 +19,22 @@
 #include <hal_data.h>
 
 const u32 _chip_type_to_odm_ic_type[] = {
-	0,
+	0, // NULL_CHIP_TYPE
+	0, // RTL8188E
+	0, // RTL8192E
 	ODM_RTL8812,
 	ODM_RTL8821,
+	0, // RTL8723B
 	ODM_RTL8814A,
-	0,
+	0, // RTL8703B
+	0, // RTL8188F
+	0, // RTL8188GTV (???)
+	0, // RTL8822B
+	0, // RTL8723D
+	0, // RTL8821C
+	0, // RTL8710B
+	0, // RTL8192F
+	0, // MAX_CHIP_TYPE
 };
 
 void rtw_hal_chip_configure(_adapter *padapter)
